### DISCUSSION TITLE
Map data export?

### DISCUSSION THREAD

**Discussion by kikiviki** [Jonathan-Greve/GuildWarsMapBrowser#16] [Ideas]
Is it possible to export the map data and terrain features as raw data points?
I would love to generate a full map like -BOB- did in 2022: 
https://www.reddit.com/r/GuildWars/comments/xo6dw5/full_presearing_navigation_mesh/

Oh, this data already exists! Turns out if you RTFM you learn so much :)

**Comment by Jonathan-Greve** [comment]
Haha yeah `_-BOB-_` is my old reddit account. That specific data cannot be extracted from GWMB at this moment. I think I have basically all the data I need to generate the navmesh in GWMB but it would take a considerable amount of work and I'll probably never get around to it. So to get the navmesh you'd have to use the [GWCA](https://github.com/gwdevhub/GWCA) framework to extract the pathing map/nav mesh data as I (_-BOB-_) did.

Please let me know if you run into any issues or if there is any other data you'd like to extract and I'll let you know whether or not it's possible as of this moment. If these things interest you, please consider joining us on the GWCA/GWToolboxpp discord server](https://discord.gg/dbFaUyyR).
